src/main/resources/metafile-schema.json

```diff
@@ -105,7 +105,7 @@
           "ocr_data": {
             "id": "#/properties/scannable_items/items/properties/ocr_data",
             "type": ["string", "null"],
-            "pattern": "^[0-9a-zA-Z=]*$"
+            "pattern": "^[0-9a-zA-Z=+/]*$"
```

Comment:
perhaps would be worth adding a "title" field here (or any other if there's a more appropriate one) do explain we expect base64 encoded string here.